cmd/subnetcmd/import_file.go

```diff
@@ -0,0 +1,284 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package subnetcmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/apmintegration"
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/spf13/cobra"
+)
+
+var (
+	overwriteImport bool
+	repoOrURL       string
+	subnetAlias     string
+	branch          string
+)
+
+// avalanche subnet import
+func newImportFileCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "file [subnetPath]",
+		Short:        "Import an existing subnet config",
+		RunE:         importSubnet,
+		SilenceUsage: true,
+		Args:         cobra.MaximumNArgs(1),
+		Long: `The subnet import command will import a subnet configuration from a file or a git repository.
+
+To import from a file, you can optionally provide the filepath as a command line argument.
```

Comment:
I may have changed this slightly in #467

Reply:
Indeed, incorporated.